docs/docs/user/getting-started/quick-start.md

```diff
@@ -48,7 +48,7 @@ redis bitnami/redis
 
 #### Port forwarding
 
-Before we deploy the pipeline, we need to forward the ports of `kafka-rest-proxy` and `kafka-connect`.
+Before we deploy the pipeline, we need to forward the ports of `kafka-rest-proxy` and `kafka-connector`.
```

Comment:
this should stay, as it is referring to the Kafka Connect server

Reply:
done